x-pack/solutions/security/plugins/security_solution/public/timelines/components/timeline/tabs/esql/index.tsx

```diff
@@ -204,7 +204,7 @@ export const DiscoverTabContent: FC<DiscoverTabContentProps> = ({ timelineId })
 
       const hasESQLUrlState = (stateContainer.appState.getState()?.query as { esql: string })?.esql;
 
-      if (stateContainer.appState.isEmptyURL() || !hasESQLUrlState) {
+      if (!stateContainer.stateStorage.get('_a') || !hasESQLUrlState) {
```

Comment:
nit, but it would be extremely helpful if we use a `constant` for the param here so if it's ever changed for any reason, that's accounted for here as well. (For backwards compatibility, that may never happen, but just a consideration if it's not too much of a lift)